### ISSUE TITLE
Support flavor in opentelekomcloud_cce_addon_v3 values

### ISSUE THREAD

**Issue by Unichron** [opentelekomcloud/terraform-provider-opentelekomcloud#2137]
It's now not possible to configure the "flavor" of the addons via this resource. The flavor field is to describe replica count and resource requests/limits.

**Comment by artem-lifshits** [comment]
Hello @Unichron could you please provide more details what should this enhancement do and for which use cases? 
The more details the better for R&D.

Thank you.

**Comment by Unichron** [comment]
Hi, sorry for the delay.

This is poorly documented in the docs but here you can find an example: https://docs.otc.t-systems.com/cloud-container-engine/api-ref/apis/add-on_management/updating_an_add-on_instance.html#request

Relevant code excerpt:
```
"values": {
    "flavor": {
        "description": "Has only one instance",
        "name": "Single",
        "replicas": 1,
        "resources": [
            {
                "limitsCpu": "100m",
                "limitsMem": "300Mi",
                "name": "metrics-server",
                "requestsCpu": "100m",
                "requestsMem": "300Mi"
            }
        ]
    }
  }
```

Currently the `opentelekomcloud_cce_addon_v3` resource supports only the `basic` and `advanced` fields in the `values` object. By adding `flavor`, which is supported by the REST API, it would be possible to manage replica count and resource requests/limits of the addons. Also note that while the documentation only mention it in the "update" section it is possible to set this during both create and update (and should be returned when read).

**Comment by artem-lifshits** [comment]
Hello @Unichron added on latest release, please check.

**Comment by eht16** [comment]
I didn't request it but also want to use it :).

On
https://registry.terraform.io/providers/opentelekomcloud/opentelekomcloud/latest
and
https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud
the latest release is still 1.34.2 :(.

**Comment by artem-lifshits** [comment]
@Unichron @eht16 my bad, it will be available on next release v1.34.3. 
I will ping you on this ticket after release.

**Comment by artem-lifshits** [comment]
@Unichron @eht16 feature is available in latest release, please check.
